recipes/colormath/meta.yaml

```diff
@@ -0,0 +1,47 @@
+
```

Comment:
If you're curious, this is why I was confused:

![image](https://cloud.githubusercontent.com/assets/465550/26213078/0916497a-3bf8-11e7-90c3-192fa1e87aff.png)


![image](https://cloud.githubusercontent.com/assets/465550/26213064/f9c69c40-3bf7-11e7-8f94-eff5f5b8c260.png)